src/main/scala/vectorpipe/osm/Element.scala

```diff
@@ -124,15 +120,14 @@ case class Member(
   ref: Long,
   role: String)
 
-/** All Elements have common attributes, and may also have additional "tags". */
-@Lenses case class ElementData(meta: ElementMeta, tagMap: Map[String, String])
```

Comment:
`ElementMeta` replaces `ElementData` in all cases. There wasn't really a reason to have them separate in the first place.